rosalind_ini6.py

```diff
@@ -1,11 +1,10 @@
 def main(input):
-    words = input.split()
     results ={}
```

Comment:
I tried to look up to see if there are dictionary comprehensions too. It looks like there are, for example 
`d = {n: n**2 for n in range(5)}`, but I don't see how to make it work here.